packages/EDA/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # EDA changelog
 
+## 1.1.4 (WIP)
+
+Implemented ANOVA.
+
+### Features
+
+* One-way ANOVA (see Top Menu > ML > Analysis of Variances (ANOVA)...)
+
 ## 1.1.3 (2023-08-24)
 
 This release is centered around enhancing user-friendliness and addressing concerns.
```

Comment:
Let's cut the BS next time :) If there is nothing specific to say but we need to put it (not a case here I think), it should be much simpler "Bug fixes and performance improvements".